e2e/cypress/integration/enterprise/guest_accounts/guest_identification_spec.js

```diff
@@ -22,8 +22,9 @@ describe('Guest Accounts', () => {
     let sysadmin;
```

Comment:
I forgot to comment in the other PR but please add `@enterprise` group metadata above so we can sort out enterprise related specs.

``// Group: @enterprise @guest_account``